src/snackbar.jsx

```diff
@@ -78,8 +131,14 @@ const Snackbar = React.createClass({
   },
 
   getInitialState() {
+    let open = this.props.open;
+
+    if (open === null) {
+      open = this.props.openOnMount;
+    }
+
     return {
-      open: this.props.openOnMount,
+      open: open,
```

Comment:
might be able to make `open` a `const` or directly assign it to this object like 
```js
{
  open: this.props.open || this.props.openOnMount,
}
```